src/tests/bulkConcepts/components/Pagination.test.js

```diff
@@ -119,6 +119,28 @@ describe('Test suite for Paginations component', () => {
     wrapper.find('.fa-angle-double-right').simulate('click');
     expect(wrapper.find('Paginations').instance().props.currentPage + 1).toEqual(2);
   });
+  it('it should go to previous page', () => {
```

Comment:
What should go?

Reply:
I have made the description clearer

Reply:
I have made adjustments to the description